client/sttp-client/src/main/scala/tapir/client/sttp/EndpointToSttpClient.scala

```diff
@@ -178,6 +180,10 @@ class EndpointToSttpClient(clientOptions: SttpClientOptions) {
             r.header(k, v, replaceExisting)
         }
         setInputParams(tail, params, paramsAsArgs, paramIndex + 1, uri, req2)
+      case EndpointIO.FixedHeader(name, value, _) +: tail =>
+        val req2 = Seq(value)
+          .foldLeft(req) { case (r, v) => r.header(name, v) }
+        setInputParams(tail, params, paramsAsArgs, paramIndex + 1, uri, req2)
```

Comment:
should it be `paramIndex + 1` as we are not consuming anything from user's input?

Reply:
I don't know, there is a very convincing message at the top of this class to not read this code, so I followed it :D

Reply:
yes, we'll make it nicer one day ;)